app/page.tsx

```diff
@@ -19,11 +19,11 @@ const Homepage = async () => {
       <Head>
         <title>Home | Everyone Plays the Same Song</title>
       </Head>
-      <div className="flex justify-center items-center h-4/5">
+      {/* <div className="flex justify-center items-center h-4/5"> */}
```

Comment:
you can just delete these. Any reason to keep the code?

Reply:
Cleaned up in last commit